soc/arm/nxp_imx/rt/CMakeLists.txt

```
@@ -25,6 +25,19 @@ if(CONFIG_PM)
 zephyr_sources_ifdef(CONFIG_SOC_SERIES_IMX_RT11XX power_rt11xx.c)
 endif()
 
+if (CONFIG_PM AND CONFIG_SOC_SERIES_IMX_RT10XX)
+zephyr_sources(power_rt10xx.c)
+zephyr_code_relocate(power_rt10xx.c ITCM_TEXT)
```

Please indent for readability:
```suggestion
  zephyr_sources(power_rt10xx.c)
  zephyr_code_relocate(power_rt10xx.c ITCM_TEXT)
```

---

Fixed in latest push